### ISSUE TITLE
argparse4s binaries for 2.11 don't exist?

### ISSUE THREAD

**Issue by xanderdunn** [mdekstrand/argparse4s#5]
The closest I can find are binaries built with Scala 2.10: http://mvnrepository.com/search?q=argparse4s

**Comment by mdekstrand** [comment]
Yeah, haven't rebuilt and published in a while. I'll try to get to that maybe this weekend.

**Comment by mdekstrand** [comment]
I'm no longer actively maintaining argparse4s. Happy to transfer it to an interested maintainer.
